release-notes/major10/10.0.0/migration.md

```diff
@@ -24,4 +24,99 @@ This page describes how to update Valtimo from the previous version to the curre
     command in the root of the implementation project:
     `ng update @angular/core@14 @angular/cli@14`
 
-       
+* **Dependency upgrades**
+
+  Scope: Frontend
+
+    In order to use the front-end libraries with upgraded dependencies, make sure to first upgrade your implementation
+    to Angular 14 by following the steps mentioned on this page.
+
+    *Redundant stylesheets*
+
+    Next, you might have to delete the following lines from the `styles` array in `angular.json` in the root of your
+    implementation:
+
+    `"node_modules/dmn-js/dist/assets/dmn-font/css/animation.css"`
+
+    `"node_modules/dmn-js/dist/assets/dmn-font/css/dmn-ie7.css"`
+
+    `"node_modules/dmn-js/dist/assets/dmn-font/css/dmn-ie7-codes.css"`
+
+    *Matching dependencies*
+
+    Then, make sure the following dependencies in `package.json` in the root of your implementation have the following
+    versions:
+
+    #### **`package.json`**
+
+    ```json
+    {
+      "dependencies": {
+        ...
+        "@auth0/angular-jwt": "^5.0.1",
+        "@mdi/font": "^7.0.96",
+        ...
+        "dmn-js": "^12.1.0",
+        "dropzone": "^6.0.0-beta.2",
+        "ngx-color-picker": "^13.0.0",
+        ...
+        "ngx-translate-multi-http-loader": "^9.0.0",
+        ...
+        "swagger-ui": "^4.15.0",
+        ...
+        "zone.js": "^0.11.8"
+      },
+      "devDependencies": {
+        ...
+        "@types/jasmine": "^4.3.0",
+        "@types/jasminewd2": "^2.0.10",
+        ...
+        "jasmine-core": "^4.4.0",
+      } 
+    }
+    ```
+  
+    Note: if your implementation has any dependencies not included in the above example, and installing dependencies
+    or building your application causes errors, make sure the versions of these dependencies match the versions
+    in this release of `valtimo-frontend-libraries`.
+
+    *Translate loader upgrade*
+
+    `ngx-translate-multi-http-loader` has been upgraded from version 7 to version 9. First of all, the import of
+    `TranslateModule` in the file `app.module.ts` in your implementation needs to be adjusted. More specifically,
```

Comment:
These two sentences feel like you're speaking to someone more than writing down instructions. Could you rephrase it so it's a bit more to the point?